sdk/storage/storage-file/src/FileClient.ts

```diff
@@ -584,4 +623,358 @@ export class FileClient extends StorageClient {
       abortSignal: aborter
     });
   }
+
+  // High Level functions
+
+  /**
+   * ONLY AVAILABLE IN BROWSERS.
+   *
+   * Uploads a browser Blob/File/ArrayBuffer/ArrayBufferView object to an Azure File.
+   *
+   * @param {Blob | ArrayBuffer | ArrayBufferView} browserData Blob, File, ArrayBuffer or ArrayBufferView
+   * @param {UploadToAzureFileOptions} [options]
+   * @returns {Promise<void>}
+   */
+  public async uploadBrowserData(
```

Comment:
One of the side effects after moving all highlevel functions into FileClient is, the generated browser or node bundle  will increase their size, because they both now include all the methods.

Reply:
We could do some trick in rollup config to remove node/browser code but that might be complicated and not worth it.